python/lsst/afw/image/utils.py

```diff
@@ -68,23 +67,6 @@ def defineFilter(name, lambdaEff, lambdaMin=np.nan, lambdaMax=np.nan, alias=[],
             Filter.defineAlias(name, a)
 
 
-class CalibNoThrow:
-    """A class intended to be used with python's with statement, to return NaNs for negative fluxes
-    instead of raising exceptions (exceptions may be raised for other purposes).
-
```

Comment:
This is the default with `PhotoCalib`, correct?
(Just to make sure I'm understanding the differences correctly.)

Reply:
Yes: `PhotoCalib` never throws due to negative fluxes.